packages/core/framework/src/config/types.ts

```diff
@@ -4,6 +4,7 @@ import {
 } from "@medusajs/types"
 
 import type { RedisOptions } from "ioredis"
+import { ConnectionOptions } from "tls"
```

Comment:
should be 'node:tls' ?

Reply:
<img width="707" alt="Screenshot 2025-02-10 at 12 46 31" src="https://github.com/user-attachments/assets/e40ec43d-4fbe-4358-ad0c-33612affc680" />

Just checked, it is from node:tls

Reply:
changed it to request directly from `node:tls`